.travis.yml

```diff
@@ -66,6 +65,10 @@ install:
   - pip install --user cpp-coveralls
 # IBM-TPM
   - wget https://download.01.org/tpm2/ibmtpm974.tar.gz
+# doxygen
+  - wget http://ftp.stack.nl/pub/users/dimitri/doxygen-1.8.14.linux.bin.tar.gz
```

Comment:
The comment says "Compiled using Ubuntu 14.04 with kernel 3.13.0 and gcc 4.8.2" that sould be ok for trusty, but we need to remember to update this as well when we update the travis dist. Could you at least add a comment here please.

Reply:
According to the CI log, the precompiled binary does not work for some reason, so I will build doxygen from source instead.